### PR TITLE
Add two exception excerises

### PR DESCRIPTION
**Pull request by momosally**
Still working on the other two assignments.

### CODE REVIEW

File: students/Sally_Shen/lesson5/MailroonPart3.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+def lineItem(name, total, numGifts, avg):
+    output = "{:<20}${:<20.2f}{:<15}${:<20.2f}\n".format(name, total, numGifts, avg)
+    return output
+
+
+def gen_thank_you_letters(a_dict):
+    dir = "thankyouLetters"
+    try:
+        os.mkdir(dir)
+    except FileExistsError:
```
Comment (nairraghav) [inline students/Sally_Shen/lesson5/MailroonPart3.py:91]: nice!

File: students/Sally_Shen/lesson5/MailroonPart3.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        pass
+    for name, amount in donors.items():
+        firstLastName = name.split()
+        fileName = firstLastName[0] + "_" + firstLastName[1] + ".txt"
+        filePath = dir + "\\" + fileName
```
Comment (nairraghav) [inline students/Sally_Shen/lesson5/MailroonPart3.py:96]: This ends up prepending dir + \\ into the filename. What you want to do is change the working directory to enter the filePath and then create the files.
For example:
os.chdir(path)

**Comment by nairraghav** [comment]
Great job Sally! I've added a comment on how to add your created files for Mailroom into the directory that you created. You can try to implement that as a part of your next mailroom

File: students/Sally_Shen/lesson5/MailroonPart3.py (hunk 2 above)
Comment (momosally) [reply students/Sally_Shen/lesson5/MailroonPart3.py:96]: Thank you, fixed in lesson 6 PR

**Pull request merged**